_partials/_caggs-types.md

```diff
@@ -1,5 +1,5 @@
-There are three main ways to make aggregation easier: materialized views,
-continuous aggregates, and real time aggregates.
+While continuous aggregates are built on PostgreSQL materialized views, there
```

Comment:
They aren#t really built upon materialized views, they just use the same API and the concept is similar. Not sure if we want to phrase it that way :)

Maybe something like "While continuous aggregates and PostgresSQL's materialized views are similar in concept, there are some important differences:"

WDYT?